release-notes/release-notes/5.7-release-notes.md

```diff
@@ -347,7 +350,9 @@ We implemented Istio mTLS support to ensure secure/TLS in-cluster communication
 - We fixed a bug that was stripping query parameters from Relative URLs if they were being used as the Application URL of the applets. In the below screenshot, the bug would have stripped out the "q=1" parameter, leaving only an Absolute URL in lieu of a Relative one.
 - We fixed an issue with the behaviour of cached calculated columns when using multi-select data types (Link, Choice, and Hierarchy) with **Change Approval** enabled. These data types should now work as expected.
 
-<figure><img src="../../.gitbook/assets/image (175).png" alt=""><figcaption></figcaption></figure>
+  <figure><img src="../../.gitbook/assets/image (175).png" alt=""><figcaption></figcaption></figure>
+
+- We resolved an issue that prevented view exports from reaching the maximum limit of 250,000 records.
```

Comment:
If we previously only exported the active page in a view, we need to call out in the release notes that this behaviour has changed.